sdk/core/azure-core-metrics-opentelemetry/pom.xml

```diff
@@ -118,12 +118,6 @@
       <version>1.14.0</version>  <!-- {x-version-update;io.opentelemetry:opentelemetry-exporter-otlp;external_dependency} -->
       <scope>test</scope>
     </dependency>
-    <dependency>
-      <groupId>io.opentelemetry</groupId>
-      <artifactId>opentelemetry-sdk-extension-autoconfigure</artifactId>
-      <version>1.14.0-alpha</version>  <!-- {x-version-update;io.opentelemetry:opentelemetry-sdk-extension-autoconfigure;external_dependency} -->
-      <scope>test</scope>
-    </dependency>
```

Comment:
just confirming that it's not possible to add a dependency only for `samples`?

Reply:
@trask 
it should be possible to create samples-only project and then it should be fine. 
To be fair, I didn't push hard enough to keep it and solve global + autoconfigure problems in tests. If you believe we absolutely need to have it in samples, I can try harder :)

Reply:
no problem, I support this PR as-is, and thx for the tip about creating a samples-only project if we need to in the future